packages/runner-ct/src/lib/state.ts

```diff
@@ -260,7 +264,7 @@ export default class State {
   }
 
   runMultiMode = async () => {
-    const eventManager = require('./event-manager')
+    const eventManager = require('./event-manager').default
```

Comment:
Kind of surprised this was working before without `default`?

Reply:
It was never tried, (it never worked) and it has nothing to do with this PR.